background.ts

```diff
@@ -856,10 +866,14 @@ class BackgroundCommandWorker implements CommandWorkerInterface {
     return diagnostics.getIdsForCategory(category);
   }
 
-  getDiagnosticChecks(category: string|null, checkID: string|null): DiagnosticsCheck[] {
+  getDiagnosticChecks(category: string|null, checkID: string|null): DiagnosticsResultGroup {
     return diagnostics.getChecks(category, checkID);
   }
 
+  runDiagnosticChecks(): Promise<DiagnosticsResultGroup> {
```

Comment:
Unless I'm mistaken, or this is for something in the future, this is dead code.

Reply:
This is used by `httpCommandServer` for `diagnosticRunChecks`. (But see above — I'll need to switch to using the HTTP API instead of IPC)

Reply:
Fixed in da002f7c1f0fb21191d9e513689e674546cd5074.

Reply:
Ah right, I didn't understand this fully.